### PR TITLE
dependency bump: marked from 3.0.8 to 4.0.1

### PR DESCRIPTION
**Pull request by humble-barnacle001**
Fixes the bump issue of dependabot in #4233 caused by bumping the major version of `marked`

### CODE REVIEW

**Review by nschonni — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,6 @@ npm-debug.log
 
 # Netlify
 .netlify
+
+#VSCode settings; makes life easier for some, tougher for most
+.vscode/
```
Comment (Trott) [inline .gitignore]: If you run VS Code or whatever creates this directory, you should probably put this in a .gitignore file in your home directory. I'd prefer we don't try to accommodate every IDE separately in our repo .gitignore.

**Review by Trott — APPROVED** [review]
LGTM. I'd prefer we pull out the `.gitignore` change, but I'm OK if we don't.

File: .gitignore (hunk 1 above)
Comment (humble-barnacle001) [reply .gitignore]: Removed in 956b1ac

**Comment by humble-barnacle001** [comment]
Bump to fix for #4249

**Comment by humble-barnacle001** [comment]
> LGTM. I'd prefer we pull out the `.gitignore` change, but I'm OK if we don't.

@Trott  Done

**Pull request merged**